tests/acceptance/core/service/assert/host.go

```diff
@@ -0,0 +1,34 @@
+package assert
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/rancher/rke2/tests/acceptance/shared/util"
+
+	. "github.com/onsi/gomega"
+)
+
+// CheckComponentCmdHost runs a command on the host and asserts that the value received contains the specified substring
+//
+// you can send multiple asserts from a cmd but all of them must be true
+//
+// need to send sKubeconfigFile
+func CheckComponentCmdHost(cmd string, asserts ...string) {
+	Eventually(func() error {
+		fmt.Printf("\nExecuting cmd: %s\n", cmd)
+		res, err := util.RunCommandHost(cmd)
+		if err != nil {
+			return fmt.Errorf("error on RunCommandHost: %v", err)
+		}
+
+		for _, assert := range asserts {
+			if !strings.Contains(res, assert) {
+				return fmt.Errorf("expected substring %q not found in result %q", assert, res)
+			}
+			fmt.Printf("\nResult: %s \n", res)
+			fmt.Printf("Matches with assert: %s \n", assert)
```

Comment:
The preferred/more idiomatic way of doing this line would be to concatenation the strings in a Println func. The use of prints is extremely heavy and unnecessary with values of type string.

```
fmt.Println(“Matches with assert: “ + assert)
```

Reply:
fixed tk you !